src/components/modals/KPIProgressModal.vue

```diff
@@ -64,6 +64,18 @@
         />
       </template>
     </form-section>
+
+    <template #subfooter>
+      <hr class="ods-hr" />
+      <i18n path="kpi.help.apiProgress" tag="p">
+        <template #apiLink>
+          <router-link :to="{ name: 'Api' }" target="_blank">
+            <span>{{ $t('general.api') }}</span>
+          </router-link>
+        </template>
+      </i18n>
+      <ProgressUpdateAPIExample :path="`kpi/${activeKpi.id}`" />
```

Comment:
Vi bruker normalt kebab-case for komponenter i templates? Sikkert litt begge deler, og kan vel som vanlig diskuteres: https://v2.vuejs.org/v2/style-guide/?redirect=true#Component-name-casing-in-templates-strongly-recommended 🤷